nbconvert/exporters/templateexporter.py

```diff
@@ -303,6 +303,9 @@ def _template_extension_default(self):
         help="""formats of raw cells to be included in this Exporter's output."""
     ).tag(config=True)
 
+    extra_template_basedirs = List(
+        help="Specify extra directories to search for templates").tag()
```

Comment:
Instead of recreating the option, you want to add the help string to the `extra_template_basedirs` assignment that's already present. By restating it here you're replacing the other assignment which is causing the errors in the tests.